src/components/EditEmpPortfolio.tsx

```diff
@@ -29,7 +30,7 @@ const EditEmpPortfolio = () => {
             submitted: true
         }
         setCookie('portfolio', obj, { path: '/' });
-        axios.post(`http://3.236.213.150:8081/portfolios/${cookies['portfolio'].id}`, { ...obj }).catch(error => {
+        axios.post(url + `/portfolios/${cookies['portfolio'].id}`, { ...obj }).catch(error => {
```

Comment:
@ddinh please store url + '/portfolios to a variable and pass here to minimize code duplication